plugins/hls-fourmolu-plugin/hls-fourmolu-plugin.cabal

```diff
@@ -41,13 +42,8 @@ library
     , text
     , transformers
 
-  if impl(ghc >= 9.0) && impl(ghc < 9.2)
-    build-depends: fourmolu ^>= 0.11
-  else 
-    build-depends: fourmolu ^>= 0.14
-
   -- fourmolu 0.9.0 fails to build on Windows CI for reasons unknown
```

Comment:
the other constraint forbids fourmolu-0.9.0 already, right?

Reply:
You're right, I missed this one. Removed.